packages/utils/src/countries.ts

```diff
@@ -140,6 +146,10 @@ export class Countries {
         const localizedCountry = {
           names,
           displayName: names[this.language],
+          countryPhonePlaceholder: {
+            national: getExampleNumber(country.countryCallingCodes[0]),
+            internation: getExampleNumber(country.countryCallingCodes[0], true, true),
```

Comment:
Don't think we need the international version right? Let's not calculate one for each country. Keep in mind this re-runs each time the app opens. 
Also, typo: `internatio` -> `international`